pkgs/applications/misc/errands/default.nix

```diff
@@ -0,0 +1,73 @@
+{ fetchFromGitHub
+, gobject-introspection
+, lib
+, libadwaita
+, python3Packages
+, wrapGAppsHook
+, meson
+, ninja
+, desktop-file-utils
+, pkg-config
+, appstream
+, libsecret
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "errands";
+  version = "45.1.3";
+
+  format = "meson";
+
+  src = fetchFromGitHub {
+    owner = "mrvladus";
+    repo = "Errands";
+    rev = "refs/tags/${version}";
+    hash = "sha256-dX0XKhOrO43J2nVuc9dMb9tscXSWQyvEIVe5Qv93EB8=";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook
+    desktop-file-utils
+    meson
+    ninja
+    pkg-config
+    appstream
+  ];
+
+  buildInputs = [
+    libadwaita
+    libsecret
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pygobject3
+    lxml
+    caldav
+  ];
+
+  nativeCheckInputs = with python3Packages; [
+    pytest
+  ];
+
+  dontWrapGApps = true;
+
+  preFixup = ''
```

Comment:
Move this after postInstall.

https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md